db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+ArtType.create([{
+  name: "visual art",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.